vendors.json

```diff
@@ -83,6 +83,11 @@
     "constant": "GOCD",
     "env": "GO_PIPELINE_LABEL"
   },
+  {
+    "name": "Heroku",
+    "constant": "HEROKU",
+    "env": { "env": "NODE", "includes": "heroku" }
```

Comment:
In the test, you check for this value: `/app/.heroku/node/bin/node`. Is that a constant? Could you check for that value explicitly instead of any value with `heroku` in it?